include/engine/routing_algorithms/routing_base_mld.hpp

```diff
@@ -539,16 +522,14 @@ UnpackedPath search(SearchEngineData<Algorithm> &engine_working_data,
             EdgeWeight subpath_weight;
             std::vector<NodeID> subpath_nodes;
             std::vector<EdgeID> subpath_edges;
-            std::tie(subpath_weight, subpath_nodes, subpath_edges) =
-                search(engine_working_data,
-                       facade,
-                       forward_heap,
-                       reverse_heap,
-                       force_loop_forward_nodes,
-                       force_loop_reverse_nodes,
-                       INVALID_EDGE_WEIGHT,
-                       sublevel,
-                       parent_cell_id);
+            std::tie(subpath_weight, subpath_nodes, subpath_edges) = search(engine_working_data,
```

Comment:
side note: structured bindings should be universally available after the compiler upgrade